t/latency_percentiles.py

```diff
@@ -309,7 +309,7 @@ def check_latencies(self, jsondata, ddir, slat=True, clat=True, tlat=True, plus=
             latencies.sort()
             ptiles = jsondata[lat+'_ns']['percentile']
 
-            for percentile in ptiles.keys():
+            for percentile in list(ptiles.keys()):
```

Comment:
I'm a bit confused as to why we need to listify the output of `keys()` - won't it already be an iterable?

Reply:
The "list()" was inserted by 2to3. Since I agree that it shouldn't be necessary, I have removed it again.